src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -37,14 +37,14 @@ import (
 )
 
 var (
-	// Changing this will cause issues if two clients with different versions are using the same account
-	CdImage = pkg.Getenv("DEFANG_CD_IMAGE", "public.ecr.aws/defang-io/cd:"+byoc.CdImageTag)
+	CdImageBase = "public.ecr.aws/defang-io/cd"
 )
 
 type ByocAws struct {
 	*byoc.ByocBaseClient
 
 	cdTasks      map[string]ecs.TaskArn
+	cdImageTag   string
```

Comment:
I like to sort these
```suggestion
	cdImageTag   string
	cdTasks      map[string]ecs.TaskArn
```